djweb/dj_comp_hist/templates/search_results.jinja2

```diff
@@ -25,34 +25,40 @@
         <br>
         <div class="form-check form-check-inline">
             <input class="form-check-input" type="checkbox" id="checkBox1"
-                   name="checkBox1" value="1">
+                   name="checkBox1" value=1 checked>
             <label class="form-check-label" for="checkBox1">1</label>
         </div>
         <div class="form-check form-check-inline">
             <input class="form-check-input" type="checkbox" id="checkBox2"
-                   name="checkBox2" value=1>
+                   name="checkBox2" value=2 checked>
             <label class="form-check-label" for="checkBox2">2</label>
         </div>
         <div class="form-check form-check-inline">
             <input class="form-check-input" type="checkbox" id="checkBox3"
-                   name="checkBox3" value=1>
+                   name="checkBox3" value=3 checked>
             <label class="form-check-label" for="checkBox3">3</label>
         </div>
     <br>
-        <label for="inputAuthor" class="col-sm-10 col-form-label">Authored by: (leave blank
+        <label for="inputAuthor" class="col-sm-10 col-form-label">Authored by: (leave Unknown
```

Comment:
Not sure we should have the person search for "Unknown"; I think it would be better to leave it blank if there is no specified author (like how Google advanced search works)